package.json

```diff
@@ -38,21 +38,21 @@
         "babel-polyfill": "^6.26.0",
         "dotenv-webpack": "^8.0.1",
         "husky": "^7.0.0",
-        "jest": "^26.4.2",
+        "jest": "^27.5.1",
         "jest-chain": "^1.1.5",
         "jest-extended": "^1.2.0",
-        "lerna": "^5.5.1",
-        "lint-staged": "^10.4.0",
-        "nx": "^14.5.10",
+        "lerna": "^6.5.0",
```

Comment:
@jim-deriv latest version is 8, why not we migrate to the latest in one shot?

Reply:
The main reason is that you removed lerna link, from bootstrap command but in version 6.5 as far as I understood its still valid and needed.

Reply:
The initial plan was to migrate to first `v7` @ali-hosseini-deriv, but then because they removed `lerna bootstrap` and I had to consider the  

I moved up one version at a time to handle as few breaking changes as possible. I remove `lerna linlk` because the [docs](https://lerna.js.org/docs/legacy-package-management) mentioned that's all we have to do for it, but for bootstrap, we had to first consider our setup in order to replace it. But I can go straight up to `v8` incase everyone is okay with it 😁

Reply:
Alright @ali-hosseini-deriv, I'll upgrade it all the way to `v8`.